__DEFINES/setup.dm

```diff
@@ -864,15 +864,17 @@ SEE_PIXELS	256
 #define RIGHT 2
 
 // for secHUDs and medHUDs and variants. The number is the location of the image on the list hud_list of humans.
-#define HEALTH_HUD          1 // a simple line rounding the mob's number health
-#define STATUS_HUD          2 // alive, dead, diseased, etc.
-#define ID_HUD              3 // the job asigned to your ID
-#define WANTED_HUD          4 // wanted, released, parroled, security status
-#define IMPLOYAL_HUD		5 // loyality implant
-#define IMPCHEM_HUD		    6 // chemical implant
-#define IMPTRACK_HUD		7 // tracking implant
-#define SPECIALROLE_HUD 	8 // AntagHUD image
-#define STATUS_HUD_OOC		9 // STATUS_HUD without virus db check for someone being ill.
+#define HEALTH_HUD          "health" // a simple line rounding the mob's number health
```

Comment:
`hud_updateflag` is broken with these changes but it's not actually used so I'm not too concerned about it.